language/ir-testsuite/tests/discovery/reconfiguration_via_network_address_rotation.mvir

```diff
@@ -39,28 +39,23 @@ main() {
 //! new-transaction
 //! sender: vivian
 import 0x0.LibraSystem;
-import 0x0.ValidatorConfig;
```

Comment:
Are all of these changes to the reconfiguration tests an accident?

Reply:
Ah they are all because of the way how branch is organized. They should not appear in the commit once the reconfiguration PR gets merged.